corehq/motech/dhis2/tests/test_entities_helpers.py

```diff
@@ -210,8 +209,12 @@ def setUp(self):
         self.supercase_config = Dhis2CaseConfig.wrap({
             'case_type': 'mother',
             'te_type_id': 'person12345',
-            'tei_id': {'case_property': 'external_id'},
-            'org_unit_id': {'case_property': 'dhis2_org_unit_id'},
+            'tei_id': {
```

Comment:
I believe where possible you have [converted](https://github.com/dimagi/commcare-hq/pull/32362/files#diff-3ce4955d53b0d6f4ed18e27077afb7a9af4ae395ada590a6ddec01fdf5fb7a6bR499) multiline statements into a single line, is there any reason to not keept these the same instead of expanding? The dense form looks as easy if not easier to read than the sparse form